docs/source/guides/distributed-services.rst

```diff
@@ -151,6 +151,65 @@ Once a dependency is declared, invoking methods on the dependent Service is simi
 
 Using ``bentoml.depends()`` is a recommended way for creating a BentoML project with distributed Services. It enhances modularity as you can develop reusable, loosely coupled Services that can be maintained and scaled independently.
 
+Depend on an external deployment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+BentoML also allows you to set an external deployment as a dependency for a Service. This means the Service can call a remote model and its exposed API endpoints. To specify an external deployment, use the ``bentoml.depends()`` function, either by providing the deployment name on BentoCloud or the URL if it's already running.
+
+.. tab-set::
+
+   .. tab-item:: Specify the Deployment name on BentoCloud
+
+      You can also pass the ``cluster`` parameter to specify the cluster where your Deployment is running.
+
+      .. code-block:: python
+
+         import bentoml
+
+         @bentoml.service
+         class MyService:
+             # `cluster` is optional if your Deployment is in a non-default cluster
+             iris = bentoml.depends(deployment="iris_classifier_x6dewa", cluster="my_cluster_name")
```

Comment:
```suggestion
             iris = bentoml.depends(deployment="iris-classifier-x6dewa", cluster="my_cluster_name")
```

Since the typical deployment name is snake-cased